src/py/flwr/client/clientapp/app.py

```diff
@@ -132,11 +132,10 @@ def run_clientapp(  # pylint: disable=R0914
             )
 
             try:
-                if fab:
-                    # Load ClientApp
-                    client_app: ClientApp = load_client_app_fn(
-                        run.fab_id, run.fab_version, fab.hash_str
-                    )
+                # Load ClientApp
+                client_app: ClientApp = load_client_app_fn(
+                    run.fab_id, run.fab_version, fab.hash_str if fab else ""
+                )
```

Comment:
@chongshenng Could you confirm that this is okay?

Reply:
Yes! This is fine, thanks for the update @charlesbvll.

Reply:
@chongshenng why are we not passing `run.fab_hash` here?